persistence-api/src/main/java/io/stargate/db/schema/SchemaBuilderImpl.java

```diff
@@ -338,6 +339,11 @@ public void indexFull() {
     indexFull = true;
   }
 
+  @DSLAction
+  public void indexCustom() {
```

Comment:
Should it also be added to the DSL grammar (the `@GenerateEmbeddedDSL` annotation on this class)?